DSCResources/MSFT_xDSCWebService/PSWSIISEndpoint.psm1

```diff
@@ -1,664 +1,663 @@
-        [String] $newVersion = "6.3.0.0"
-
-        )
-                
-    $webconfig = Join-Path $path "web.config"
-
-    if (Test-Path $webconfig)
-    {
-        $xml = [xml](get-content $webconfig)
-
-        if(-not($xml.get_DocumentElement().runtime))
-        {
-            # Create the <runtime> section
-            $runtimeSetting = $xml.CreateElement("runtime")
-
-            # Create the <assemblyBinding> section
-            $assemblyBindingSetting = $xml.CreateElement("assemblyBinding")
-            $xmlnsAttribute = $xml.CreateAttribute("xmlns")
-            $xmlnsAttribute.Value = "urn:schemas-microsoft-com:asm.v1"
-            $assemblyBindingSetting.Attributes.Append($xmlnsAttribute)
-
-            # The <assemblyBinding> section goes inside <runtime>
-            $null = $runtimeSetting.AppendChild($assemblyBindingSetting)
-
-            # Create the <dependentAssembly> section
-            $dependentAssemblySetting = $xml.CreateElement("dependentAssembly")
-
-            #The <dependentAssembly> section goes inside <assemblyBinding>
-            $null = $assemblyBindingSetting.AppendChild($dependentAssemblySetting)
-
-            # Create the <assemblyIdentity> section
-            $assemblyIdentitySetting = $xml.CreateElement("assemblyIdentity")
-            $nameAttribute = $xml.CreateAttribute("name")
-            $nameAttribute.Value = "microsoft.isam.esent.interop"
-            $publicKeyTokenAttribute = $xml.CreateAttribute("publicKeyToken")
-            $publicKeyTokenAttribute.Value = "31bf3856ad364e35"
-            $null = $assemblyIdentitySetting.Attributes.Append($nameAttribute)
-            $null = $assemblyIdentitySetting.Attributes.Append($publicKeyTokenAttribute)
-
-            # <assemblyIdentity> section goes inside <dependentAssembly>
-            $dependentAssemblySetting.AppendChild($assemblyIdentitySetting)
-
-            # Create the <bindingRedirect> section
-            $bindingRedirectSetting = $xml.CreateElement("bindingRedirect")
-            $oldVersionAttribute = $xml.CreateAttribute("oldVersion")
-            $newVersionAttribute = $xml.CreateAttribute("newVersion")
-            $oldVersionAttribute.Value = $oldVersion
-            $newVersionAttribute.Value = $newVersion
-            $null = $bindingRedirectSetting.Attributes.Append($oldVersionAttribute)
-            $null = $bindingRedirectSetting.Attributes.Append($newVersionAttribute)
-
-            # The <bindingRedirect> section goes inside <dependentAssembly> section
-            $dependentAssemblySetting.AppendChild($bindingRedirectSetting)
-
-            # The <runtime> section goes inside <Configuration> section
-            $xml.configuration.AppendChild($runtimeSetting)
-
-            $xml.Save($webconfig) 
-        }
-    }
-}
-
-Export-ModuleMember -function New-PSWSEndpoint, Set-AppSettingsInWebconfig, Set-BindingRedirectSettingInWebConfig
-
+﻿# This module file contains a utility to perform PSWS IIS Endpoint setup

```

Comment:
I suspect this change caused by line endings change. We should consider adding test for that to common tests.